src/components/CodeReminderModal/index.jsx

```diff
@@ -137,9 +131,12 @@ class CodeReminderModal extends React.Component {
           )}
           {isBulkRemind && data.selectedCodes && (
             <React.Fragment>
-              {data.selectedCodes.length > 0 && <p className="bulk-selected-codes">Selected Codes: {data.selectedCodes.length}</p>}
+              <p className="bulk-selected-codes">
+                Selected Codes: {data.selectedCodes.length > 0 ? data.selectedCodes.length : 'All'}
```

Comment:
Good call on saying "All" here when all codes are selected. This was something I considered adding as well. I think we should do this for the other modals (i.e., assign/revoke) in a separate PR. I'll file a ticket for it.

Reply:
Alternatively, I wonder if it makes more sense to state the actual number of codes instead of just saying "All". Like we do when the user manually selects codes across all pages, the `CodeAssignmentModal` will say the total number of codes across all pages.

Reply:
I chatted with Jennifer about showing "All" versus the actual number. We decided that showing the actual number of codes is the approach we would want to take to reduce ambiguity around how many "All" is.

To get the total number, you can map the coupon details table data to the `CodeReminderModal` container similar to the [`CodeAssignmentModal` container](https://github.com/edx/edx-portal/blob/master/src/containers/CodeAssignmentModal/index.jsx#L7).

Reply:
Ok 👍 

I just looked at the code for codeAssignment selection handling - PR https://github.com/edx/edx-portal/pull/163 , I guess, we should add it to other filter views too at-least Unredeemed (may be in a separate ticket). That's a nice UX addition (making things more explicit for assign-all/remind-all/revoke-all). My just concern is bulk "revoke", which would need key value pair ...

Reply:
@mushtaqak If we're supporting bulk revoke in the API, I would assume we would need to do similar work as you did for the remind all work, where we make the `assignments` field option and revoke all if `assignments` is not provided?

Reply:
makes sense, once revoke-all is implemented we can use (select-all) UI for unredeemed and partially-redeemed toggles as well. I will document this in a separate JIRA story.